src/functTest/groovy/com/bmuschko/gradle/docker/AbstractFunctionalTest.groovy

```diff
@@ -34,6 +45,20 @@ abstract class AbstractFunctionalTest extends Specification {
 
     String dockerServerUrl
 
+    @Shared
+    DockerClientConfig config = DefaultDockerClientConfig.createDefaultConfigBuilder()
+        .withDockerHost("unix:///var/run/docker.sock")
```

Comment:
This is why I suggested to use the AbstractDockerTask and grab the client from there. It removes the need to do this as well as define a docker-java version in yet another place.